test/integration/integration_test.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 const (
-	datafile   = "/tmp/datafile"
+	datafile   = "/tmp/datafilexml.xml"
```

Comment:
What's this change do?

Reply:
probably not needed, i think. I should revert this.

Reply:
reverted